modules/claims_api/spec/sidekiq/evidence_waiver_builder_job_spec.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'bgs_service/benefit_claim_service'
+require 'common/file_helpers'
 
-RSpec.describe ClaimsApi::EvidenceWaiverBuilderJob, type: :job do
+RSpec.describe ClaimsApi::EvidenceWaiverBuilderJob, type: :job, use_cassette: 'claims_api/bd/upload' do
   subject { described_class }
```

Comment:
@stiehlrod  I do not look like this cassette is used here, as well as neither of the requires on lines 4 & 5